Prace_domowe/Praca_domowa1/Grupa4/ulasik_kinga/PD1_forest_fires_EDA.ipynb

```diff
@@ -0,0 +1,1825 @@
+{
```

Comment:
<p>"wykazywać outliery" to całkiem zabawne</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>